SchoolExplorer.Application/Mappings/SchoolDistrictProfile.cs

```diff
@@ -0,0 +1,18 @@
+﻿using AutoMapper;
+using SchoolExplorer.Application.Dtos;
+using SchoolExplorer.Domain.Entities;
+
+namespace SchoolExplorer.Application.Mappings
+{
+	public class SchoolDistrictProfile : Profile
+	{
+		public SchoolDistrictProfile()
+		{
+			CreateMap<CreateSchoolDistrictDto, SchoolDistrictDto>();
+			CreateMap<SchoolDistrictDto, CreateSchoolDistrictDto>();
+			CreateMap<CreateSchoolDistrictDto, SchoolDistrict>();
+			CreateMap<SchoolDistrict, SchoolDistrictDto>();
```

Comment:
If you have actual use cases then you can use reverse map once you understand what it is and how it’s different